spec/active_support/cache/database_store_spec.rb

```diff
@@ -107,6 +107,21 @@
     expect { subject.read("#{key}x") }.to raise_error(ArgumentError, /exceeds the length limit/)
   end
 
+  it 'write_multi to insert records with one insert' do
+    insert_queries = []
+    ActiveSupport::Notifications.subscribe('sql.active_record') do |_name, _start, _finish, _id, payload|
+      insert_queries << payload[:sql] if payload[:sql].start_with?('INSERT INTO')
```

Comment:
I'm not sure it's the best way to check this. I would prefer just to do:

```rb
subject.write_multi(values)
expect(subject.read_multi('test0', 'test1', 'test2')).to eq(values)

expect(Model.where(key: values.keys).all.map(&:to_hash)).to match_array([
  ...
])
```

Reply:
I'm just counting number of INSERT queries here, because my goal is to have only one :)

Reply:
sure, but I would also like to check that the written records are "complete", i.e. contain all the expected attributes, hence `match_array` which supports pattern matching

Reply:
I don't think this works the way you think this works.

<img width="1510" alt="Screenshot 2023-09-19 at 16 34 18" src="https://github.com/bsm/activesupport-cache-database/assets/544377/78312db7-3b49-4277-bd32-190f3ce0f3a9">